### PR TITLE
Extend technical validation (EXPOSUREAPP-8705)

### PR DESCRIPTION
**Pull request by mtwalli**
- Extend technical validation to check signature validation 
Screenshot 2021-07-27 at 17 52 54 Screenshot 2021-07-27 at 17 54 01

### CODE REVIEW

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/validation/core/DccValidator.kt [hunk 1]
```diff
@@ -39,13 +42,23 @@ class DccValidator @Inject constructor(
         return DccValidation(
             userInput = userInput,
             validatedAt = timeStamper.nowUTC,
+            signatureCheckPassed = signatureCheckPassed,
+            certificateExpiredAt = certificate.header.expiresAt.toLocalDateTime(DateTimeZone.UTC),
```
Comment (chiljamgossow) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/validation/core/DccValidator.kt:46]: Shouldn't this be converted using user tz as it is shown to the user? Or is there a requirement to use utc?

Comment (mtwalli) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/validation/core/DccValidator.kt]: Make sense

**Review by NataliaLemmerth — DISMISSED** [review]
UA approved

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/validation/core/DccValidation.kt [hunk 2]
```diff
@@ -13,6 +13,8 @@ import org.joda.time.LocalDateTime
 data class DccValidation(
     val userInput: ValidationUserInput,
     val validatedAt: Instant,
+    val signatureCheckPassed: Boolean,
+    val certificateExpiredAt: LocalDateTime,
```
Comment (chiljamgossow) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/validation/core/DccValidation.kt:17]: I would have passed the Instant here and converted in the UI. But that's up to you

**Review by chiljamgossow — DISMISSED** [review]

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/validation/core/DccValidation.kt (hunk 2 above)
Comment (chiljamgossow) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/validation/core/DccValidation.kt:17]: Actually, I would have taken the date from the certificate, and not have added it to the validation result.

Comment (mtwalli) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/validation/core/DccValidation.kt:17]: You are right, totally forgot that, it should be aligned with other related certificate information in other view holders

**Review by NataliaLemmerth — APPROVED** [review]
UA approved

**Review by chiljamgossow — APPROVED** [review]
LGTM

**Review by axelherbstreith — APPROVED** [review]

**Pull request merged**